### PR TITLE
Fix README links

### PR DESCRIPTION
**Pull request by user512**
Fix broken development setup links.

### CODE REVIEW

**Review by anaulin — APPROVED** [review]
Ah, great, I was meaning to fix that! 🙏🏼

**Comment by user512** [comment]
I think we can link `About The Zinc Collective` to https://www.zinc.coop/, thought?

File: README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ Spaces where anyone can (play || work || grow || be). 🎲 👩‍💻 🌱 🛋
...
   - [Help and Support](#help-and-support)
-- [Developing Convene](#developing-convene)
-  - [Architecture](#architecture)
-  - [Developer setup and documentation](#developer-setup-and-documentation)
-- [Design](#design)
+- [Developing Convene](CONTRIBUTING.md)
+  - [Architecture](CONTRIBUTING.md#11-architecture)
+  - [Developer setup and documentation](CONTRIBUTING.md#2-machine-setup)
+- [Design](CONTRIBUTING.md#12-design)
 - [About The Zinc Collective](#the-zinc-collective)
```
Comment (user512) [inline README.md:16]:
```suggestion
- [About The Zinc Collective](https://www.zinc.coop/)
```

Comment (user512) [inline README.md:16]:
```suggestion
- [About The Zinc Collective](#about-the-zinc-collective)
```

Comment (anaulin) [reply README.md]: 👍

**Pull request merged**